### ISSUE TITLE
Swiping is difficult on large audiobook

### ISSUE THREAD

**Issue by jammekod** [GrakovNe/lissen-android#94]
Hey,

First off, I’m glad to see this project so active! I've been using the app a lot and even started contributing with translations—really enjoying it so far.

One thing I’ve noticed is that swiping down to navigate through audiobooks with a lot of chapters can be a bit tricky and time-consuming. It might be worth considering a scroll bar, faster swipe responsiveness, or some other way to make chapter navigation smoother.

Just wanted to share this as a suggestion—thanks for all the great work!

https://github.com/user-attachments/assets/b93ae578-3c88-470c-acea-13b66699c2dc

**Comment by jammekod** [comment]
And how do I view the number of hours per book? As shown in the homepage? ![Screenshot_20241222-155207.png](https://github.com/user-attachments/assets/aaf2a6e8-a92d-498c-8e99-41807f023843)

**Comment by GrakovNe** [comment]
Thank you for the report!

Regarding the cloud icon and timings, I want to clarify that the screenshot is outdated, and I’ll update it soon. The cloud icon has been moved to the player screen, where it now offers more complex functionality with various download options. The text displaying playing times no longer exists; I plan to rethink how to reintroduce it without disrupting the design.

If you swipe up the playing queue on the player screen, you can scroll through the episodes and click the one you want to move it in a single action. Alternatively, you can tap the “Episodes” button (the first one in the bottom row on the player screen) instead of swiping.

**Comment by jammekod** [comment]
Thanks for the quick reply! I wanted to show that scrolling is not fluid :/ I would be better if it was easier to scroll on large lists.

**Comment by GrakovNe** [comment]
Yes, I understand what you mean. You're talking about inertial scrolling, and it will be implemented in the future.

Thank you for the report!
